addons/test_mass_mailing/tests/test_blacklist_behavior.py

```diff
@@ -91,3 +91,38 @@ def _test_mailing_bounce_w_auto_bl(self, bounce_base_values):
             [{'email': 'test.record.00@test.example.com', 'trace_status': 'cancel', 'failure_type': 'mail_bl'}],
             new_mailing, target, check_mail=True
         )
+
+
+@tagged('mail_blacklist')
+class TestBlackListAlwaysApplied(common.TestMassMailCommon):
```

Comment:
In 'test_mass_mailing/tests/test_mailing.py' you have a test 'test_mailing_w_blacklist' . Can't we move a new test after that one (like test_mailing_w_blacklist_nomixin or something like that) ? Just to keep things ranged :socks: :)